examples/README.md

```diff
@@ -34,6 +34,22 @@ MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run presigned-url-example
 
 Example Code: [presigned-url-example.ts](presigned-url-example.ts)
 
+## Running the Dictionary Example
+
+This example demonstrates how to use the dictionary data type.
+
+- Node version 16 or higher is required
```

Comment:
nit: this isn't your fault but while you're in here can we consolidate the requirements stuff up to one section at the top?  and we actually only require node 14 or higher.

Reply:
also feel free to do this as a follow-up PR now that i think about it.  whichever you prefer.